soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -251,85 +251,85 @@ void InitTrickNames() {
-     Text{"Gerudo Tunic", "Tunique Gerudo", "Sayo gerudo"},
-     Text{"Magic Armor", "Armure Magique", "Túnica Goron"},
+     Text{"Gerudo Top", "Corsage de damoiselle", "Pechera gerudo"},
+     Text{"Flamebreaker Armor", "Armure de pierre", "	Armadura ignífuga"},
      Text{"Red Mail", "Habits Rouges", "Ropas rojas"}};
   trickNameTable[GI_TUNIC_ZORA] = {
-     Text{"Rito Tunic", "Tunique Rito", "Sayo rito"},
+     Text{"Mermaid Suit", "Costume de sirène", "Costume de sirène"},
      Text{"Zora Armor", "Armure Zora", "Túnica Zora"},
      Text{"Blue Mail", "Habits Bleus", "Ropas azules"}};
   trickNameTable[GI_BOOTS_IRON] = {
-     Text{"Iron Hoofs", "Patins de Plomb", "Botas férreas"},
-     Text{"Snow Boots", "Bottes de Neige", "Botas de nieve"},
+     Text{"Red Boots", "Bottes rouges", "Botas rojas"},
+     Text{"Zora Greaves", "Zora Greaves", "Zora Greaves"}, //machine translation failed?
      Text{"Boots of Power", "Bottes de Puissance", "Botas de plomo"}};
   trickNameTable[GI_BOOTS_HOVER] = {
-     Text{"Hover Hoofs", "Patins des airs", "Botas flotadoras"},
+     Text{"Golden Boots", "Bottes dorées", "Botas de Oro"},
      Text{"Pegasus Boots", "Bottes pégase", "Botas de Pegaso"},
      Text{"Boots of Speed", "Bottes de vitesse", "Botas del desierto"}};
   trickNameTable[GI_WEIRD_EGG] = {
      Text{"Poached Egg", "Oeuf à la coque", "Huevo pasado"},
      Text{"Lon Lon Egg", "Oeuf Lon Lon", "Huevo Lon Lon"},
      Text{"Zora Egg", "Oeuf Zora", "Huevo Zora"}};
   trickNameTable[GI_LETTER_ZELDA] = {
-     Text{"Ruto's Letter", "Lettre de Ruto", "Carta de Ruto"},
-     Text{"Royal Letter", "Lettre Eoyale", "Carta para Kafei"},
-     Text{"Zelda's Business Card", "Carte d'affaires de Zelda", "Carta"}};
+     Text{"Letter to Kafei", "Lettre pour Kafei", "Carta para Kafei "},
+     Text{"Goat's Letter", "Lettre de chèvre", "Carta de la Cabra"},
+     Text{"Maggie's Letter", "Lettre de Maggy", "Carta de Dolores"}};
   trickNameTable[GI_BOOMERANG] = {
-     Text{"Prank Fetch Toy", "Inséparable Bâtonnet", "Bumerang"},
+     Text{"Banana", "Banane", "Plátano"},
      Text{"Gale Boomerang", "Boomerang Tornade", "Bumerán tornado"},
      Text{"Magic Boomerang", "Boomerang Magique", "Bumerán mágico"}};
   trickNameTable[GI_LENS] = {
      Text{"Sheikah-leidoscope", "Sheikah-léidoscope", "Monóculo de la Verdad"},
      Text{"Sheikah Sensor", "Sonar Sheikah", "Sensor Sheikah"},
-     Text{"Magnifying Lens", "Loupe", "Lente Aumentadora"}};
+     Text{"Crystal of Vision", "Cristal de vision", "Cristal de Visión"}};
   trickNameTable[GI_HAMMER] = {
      Text{"Goron Gavel", "Masse Perforatrice", "Mazo Goron"},
      Text{"Magic Hammer", "Marteau Magique", "Martillo mágico"},
      Text{"Skull Hammer", "Maillet Ressort", "Martillo de hierro"}};
   trickNameTable[GI_STONE_OF_AGONY] = {
-     Text{"Shard of Agahnim", "Fragment d'Agahnim", "Piedra de Agahnim"},
+     Text{"Cave Charm", "Charme de grotte", "Amuleto de la cueva"},
```

Comment:
This was a cadance of hyrule refernce which i gave priority as it did the same thing as stone of agony https://zeldawiki.wiki/wiki/Cave_Charm. Also if agahnim is kept it should be stone because of the stone/shard difference (the old one was from 3ds)